pandas/core/arrays/base.py

```diff
@@ -516,7 +518,15 @@ def nbytes(self) -> int:
     # Additional Methods
     # ------------------------------------------------------------------------
 
-    def astype(self, dtype, copy=True):
+    @overload
+    def astype(self, dtype: NpDtype, copy: bool = True) -> np.ndarray:
```

Comment:
The defaults are not needed in the overloads

```suggestion
    def astype(self, dtype: NpDtype, copy: bool = ...) -> np.ndarray:
```

Reply:
done in next commit